src/main/java/bio/terra/service/snapshot/flight/create/CreateSnapshotFireStoreDataStep.java

```diff
@@ -72,6 +76,11 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
                             mapTable.getFromTable().getName(),
                             mapTable.getFromTable().getId().toString(),
                             mapColumn.getFromColumn());
+                        List<String> uniqueRefIds = refIds.stream().distinct().collect(Collectors.toList());
```

Comment:
Hmm... I would have put this inside of the dependencyDao and not here. It'll do for now.

Reply:
Yeah, I was thinking that we may want to reuse the distincted values in `addFilesToSnapshot`